src/subscription.ts

```diff
@@ -726,9 +815,12 @@ export class Subscription extends EventEmitter {
     };
 
     if (is.string(snapshot)) {
```

Comment:
Just switch this to `typeof snapshot === 'string'` so you can drop the `toString` below :)

Reply:
done